src/author/author.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { CreateAuthorDto } from './dto/create-author.dto';
+import { UpdateAuthorDto } from './dto/update-author.dto';
+import { AuthorInterface } from './interfaces/author.interface';
+import { FindOneAuthorInterface } from './interfaces/find-one-author.interface';
+import { AuthorRepository } from './repository/author.repository';
+
+@Injectable()
+export class AuthorService {
+  constructor(private authorRepository: AuthorRepository) {}
+
+  create(createAuthorDto: CreateAuthorDto): AuthorInterface {
+    return this.authorRepository.create(createAuthorDto);
+  }
+
+  findAll(): AuthorInterface[] {
+    return this.authorRepository.findAll();
+  }
+
+  findOne(id: number): FindOneAuthorInterface {
```

Comment:
Delete Index While You Wull Return Just Add delete author.index while you will return author

Reply:
?